### PR TITLE
Add setting to suppress unknown attribute warnings

### LINKED ISSUE #1924: Option to auto-remove unknown attributes or remove ignoring message spam.
Paper dev 597 (Paper 1.13.2)

My console seems to be spamming this warning repeatedly (50+ times/sec).

```
[15:53:39] [Server thread/WARN]: Ignoring unknown attribute 'generic.rideSpeed'
[15:53:39] [Server thread/WARN]: Ignoring unknown attribute 'generic.rideMaxY'
```

This attribute was added via the plugin Ridables by @BillyGalbreath.
https://github.com/BillyGalbreath/Ridables/blob/8ecb377fc74f454411610a004e6a335edbddcf3f/src/main/java/net/pl3x/bukkit/ridables/entity/RidableType.java#L148
https://www.spigotmc.org/resources/1-13-2-ridables-ride-and-control-any-mob-with-full-wasd-control.58985/

I removed the plugin for testing purposes due to it overriding some vanilla right click interactions.

Would it be possible to add an option to automatically strip (remove) unknown attributes instead of constantly complaining? Or alternatively, add an option to disable the check / warning? I have no way to bulk edit all player.dat files to remove this attribute, or even worse, on all living entities (mobs) that it's applied retrospectively on.

### PR DESCRIPTION
**Pull request by BillyGalbreath**
Fixes #1924

### CODE REVIEW

File: Spigot-Server-Patches/0438-Add-setting-to-suppress-unknown-attribute-warnings.patch [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+diff --git a/src/main/java/net/minecraft/server/GenericAttributes.java b/src/main/java/net/minecraft/server/GenericAttributes.java
+index 90367a53..7358e412 100644
+--- a/src/main/java/net/minecraft/server/GenericAttributes.java
++++ b/src/main/java/net/minecraft/server/GenericAttributes.java
+@@ -4,6 +4,8 @@ import java.util.Collection;
+ import java.util.Iterator;
+ import java.util.UUID;
+ import javax.annotation.Nullable;
++
++import com.destroystokyo.paper.PaperConfig;
```
Comment (Spottedleaf) [inline Spigot-Server-Patches/0438-Add-setting-to-suppress-unknown-attribute-warnings.patch]: import?

**Comment by electronicboy** [comment]
From the previous time this was raised, I was skeptical on the whole "providing means to disable legit warnings", there are cases where they can be annoying, but overall dropping of them is highly questionable to me

**Comment by nossr50** [comment]
I share the same opinion as @electronicboy although I'm not a contributor so you can ignore me

**Comment by BillyGalbreath** [comment]
Closing due to lack of interest

**Pull request closed without merging**